src/Components/SubHeader/SubHeader.js

```diff
@@ -0,0 +1,55 @@
+import "./SubHeader.css";
+import ratingIcon from "../../assets/icons/star.svg";
+import MenuFilter from "./Restaurant/MenuFilter";
+import BreadCrumb from "../../commonComponents/BreadCrumb";
+import { breadCrumbConfig, restaurantConfig } from "./constant/config";
+import Offers from "./Restaurant/Offers";
+import Feedback from "./Restaurant/Feedback";
+
+const SubHeader = function () {
+  const {
+    thumbnail,
+    name,
+    category,
+    area,
+    offers,
+    aggregateRating: { ratingCount, ratingValue },
+    deliveryTime,
+    priceRange: { cost, count },
+  } = restaurantConfig;
+
+  return (
+    <>
+      <BreadCrumb items={breadCrumbConfig} />
+      <section className="restaurant">
+        <div className="restaurant-info flex justify-content-space-around align-items-center">
+          <img src={thumbnail} alt="thumbnail" className="thumbnail" />
+          <div className="restaurant-about flex-column justify-content-center">
+            <h3>{name}</h3>
+            <p>
+              <span>Closes Soon</span>
+              &nbsp;
+              <span>{category}</span>
+            </p>
+            <p>{area}</p>
+            <div className="flex align-items-center">
+              <Feedback
+                icon={ratingIcon}
+                title={ratingValue.toFixed(1)}
+                subtitle={`${ratingCount}+ ratings`}
+              />
+              <Feedback
```

Comment:
Feedback components could be rendered using map function on data which would be stored on some config file

Reply:
Ya i thought of it, the structure is similar but the output data is very different, that's why I didn't use map